DevDynamo.Web/Areas/ApiV1/Controllers/SystemController.cs

```diff
@@ -0,0 +1,34 @@
+using DevDynamo.Models;
+using DevDynamo.Web.Areas.ApiV1.Models;
+using DevDynamo.Web.Data;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Reflection;
+
+namespace DevDynamo.Web.Areas.ApiV1.Controllers
+{
+    [Route("api/v1/[controller]")]
+    [ApiController]
+    public class SystemController : AppControllerBase
+    {
+        [HttpGet]
+        public ActionResult<SystemResponse> GetVersionSystem()
+        {
+         
+            var v = GetType().Assembly.GetName().Version.ToString();
+            var envi = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
```

Comment:
We should use the `EnvironmentName` of the `IWebHostEnvironment` injected into the controller. 

Access to the environment variable might not be correct in every case.